DAO/emerald city dao case study.md

```diff
@@ -1,45 +1,55 @@
 ![PROJECT NAME](images/cdao.jpg)
 * :house: **Homepage:** [Emerald City DAO](https://ecdao.org)
-* :bird: **Emerald City DAO Twitter:** [@Emerald_DAO](https://twitter.com/emerald_dao)
-* :bird: **Jacob's Twitter:** [@Jacobmtucker](https://twitter.com/jacobmtucker)
-* :computer: **Discord:** [Discord.gg/EmeraldCity](https://discord.gg/emeraldcity)
+* :bird: **Emerald City DAO Twitter:** [@emerald_dao](https://twitter.com/emerald_dao)
+* :bird: **Jacob's Twitter:** [@jacobmtucker](https://twitter.com/jacobmtucker)
+* :computer: **Discord:** [discord.gg/emeraldcity](https://discord.gg/emeraldcity)
 
 ## :dizzy: What is Emerald City DAO?
-Emerald City is the first DAO on Flow, founded in early November of 2021. We are doing a magnitude of things including educating the Flow ecosystem on Flow & Cadence development, developing tooling & products for DAOs/projects on Flow, and figuring out crucial (fair and inclusive) governance strategies for DAOs on Flow.
-
+Emerald City is the first DAO on Flow, founded in early November of 2021. We have formed 3 Guilds to accomplish our goals:
+1. Education Guild - Educating the Flow ecosystem on Flow & Cadence development
+2. Building Guild - Developing tooling & products for DAOs/projects on Flow
+3. Governance Guild - Architecting and building crucial (fair and inclusive) governance strategies for DAOs on Flow.
 
 ## :pencil2: What makes Emerald City DAO unique?
 **1. Educate the Flow ecosystem** 
 
-This is unique simply because education on Flow is so rare, in fact mostly non-existant. Outside of my (Jacob's) YouTube channel, there are no Cadence educational resources to get people interested in the language. Thus, one of our goals at Emerald City is to educate developers and onboard people into the Flow ecosystem. We are doing that with our "Emerald Academy", [which you can find here](https://academy.ecdao.org).
+This is unique simply because education on Flow is so rare, in fact mostly non-existant. Outside of my (Jacob's) YouTube channel, there are no Cadence educational resources to get people interested in the language. Thus, one of our goals at Emerald City is to educate developers and onboard people into the Flow ecosystem. We are doing that with our "Emerald Academy", [which you can find here](https://academy.ecdao.org). We host bootcamps nearly every month to engage the Flow ecosystem and help teach everyone about the wonders of Flow and Cadence. In addition, all of our graduates have first-class access to open developer positions and work opportunities in the space.
```

Comment:
> This is unique simply because education on Flow is so rare, in fact mostly non-existent beyond the variety tools that the Flow team has made available. Outside of my (Jacob's) YouTube channel, there are no Cadence educational resources to get people interested in the language that aren't published by Flow core contributors directly.